src/components/NavBar/SearchBar.tsx

```diff
@@ -316,6 +320,8 @@ export const SearchBar = () => {
     setSearchValue('')
   }, [pathname])
 
+  const placeholderText = phase1Flag === NftVariant.Enabled ? t`Search tokens and NFT collections` : t`Search tokens`
```

Comment:
you would make two separate <Trans> components here instead of importing t

Reply:
I can't set placeholder text with a JSX Element. Using `t` was previously used for placeholder text in `src/components/SearchModal/CurrencySearch.tsx` and `src/components/SearchModal/ManageLists.tsx` and `src/pages/CreateProposal/ProposalEditor.tsx`

Reply:
check what i did in `SearchBar` in TokenTable?

Reply:
https://github.com/Uniswap/interface/blob/main/src/components/Tokens/TokenTable/SearchBar.tsx#L62

Reply:
@vm why do you prefer not using `t` here?

Reply:
Also curious about not using `t`. Tried to implement the logic in the above file which causes a lot of bloat due to the feature flag conditional. Here's the code:
```
const SearchInput = () => {
    return phase1Flag === NftVariant.Enabled ? (
      <Trans
        render={({ translation }) => (
          <Box
            as="input"
            placeholder={`${translation}`}
            width="full"
            onChange={(event: ChangeEvent<HTMLInputElement>) => {
              !isOpen && toggleOpen()
              setSearchValue(event.target.value)
            }}
            className={styles.searchBarInput}
            value={searchValue}
          />
        )}
      >
        Search tokens and NFT collections
      </Trans>
    ) : (
      <Trans
        render={({ translation }) => (
          <Box
            as="input"
            placeholder={`${translation}`}
            width="120"
            onChange={(event: ChangeEvent<HTMLInputElement>) => {
              !isOpen && toggleOpen()
              setSearchValue(event.target.value)
            }}
            className={styles.searchBarInput}
            value={searchValue}
          />
        )}
      >
        Search tokens
      </Trans>
    )
  }

Reply:
I found an unrestricted replacement for t looking through the [lingui docs](https://lingui.js.org/ref/macro.html): `i18n`

Reply:
was just trying to avoid a restricted import. if it's actually okay we should unrestrict in the rest of the app imo.